README-ruRU.md

```diff
@@ -2544,7 +2544,7 @@
 
 * <a name="alias-method"></a>
   Всегда применяйте `alias_method` для определения алиасов методов модулей,
-  классов или синглетных классов во время выполнения, так как `alias`
+  классов или  синглтон-классов во время выполнения, так как `alias`
```

Comment:
I don't see your motivation, it's quite wide spread term in the Russian community, see e.g. the translation of Hal Fultons book.